src/Controller/TideApiController.php

```diff
@@ -155,29 +155,35 @@ public static function create(ContainerInterface $container) {
   public function getRoute(Request $request) {
 
     $code = Response::HTTP_NOT_FOUND;
-    $json_response = [
-      'links' => [
-        'self' => Url::fromRoute('tide_api.jsonapi.alias')->setAbsolute()->toString(),
-      ],
-    ];
-    $json_response['errors'] = [$this->t('Path not found.')];
     $entity = NULL;
 
     $path = $request->query->get('path');
     $site = $request->query->get('site');
 
+    $json_response = [
+      'data' => [
+        "type" => 'route',
+        'links' => [
+          'self' => Url::fromRoute('tide_api.jsonapi.alias')->setAbsolute()->toString(),
+        ],
+      ],
+      'errors' => [$this->t('Path not found.')],
+    ];
+
     try {
       if ($path) {
         $cid = 'tide_api:route:path:' . hash('sha256', $path . $site);
 
+        $json_response['data']['id'] = $cid;
```

Comment:
@enzolutions cache ID should not be exposed as `id`. Please use the returned entity UUID instead.

Reply:
Thanks, @sonnykt for the suggestion I will implement the change